lib/Server.js

```diff
@@ -202,7 +202,7 @@ class Server {
       }
 
       this.sendMessage(
-        this.webSocketServer.webSocketConnections,
+        this.webSocketServer && this.webSocketServer.webSocketConnections,
```

Comment:
No need to check `this.webSocketServer`, it will be always non null-able here, better rewrite tests